core/sodasql/cli/cli.py

```diff
@@ -493,19 +493,22 @@ def scan(scan_yml_file: str, warehouse_yml_file: str, variables: tuple, time: st
         sys.exit(1)
 
 
-@main.command(short_help="Ingest test information from different tools")
-@click.argument(
-    "tool",
-    required=True,
-    type=click.Choice(["dbt"]),
-)
+@main.group()
 @click.option(
     "--warehouse-yml-file",
     help="The warehouse yml file.",
     required=True,
 )
+def ingest():
+    """Commands for ingesting test information from other tools."""
+
+
+@ingest.command(
+    name="dbt",
+    short_help="Ingest test information from different tools"
+)
 @click.option(
-    "--dbt-artifacts",
+    "--artifacts-directory",
```

Comment:
I'm not sure I buy into the rename here but maybe I need to check the following: is it the case that all if we were to use a different sub-command the set of options would be different and therefore we do not need to make the options bear a tool-specific name?

One thing though, is that since we've already released this with the old names, not offering support for the old name would be a breaking change, which we don't want. I'm pretty sure there's a way to make click accept different versions of an option that map onto the same one right? If so, would we temporarily allow for that? Then What I would do is throw a deprecation warning that `--dbt-` CLI options will stop being supported in version x and make sure that we deprecate it in said version.

Reply:
> is it the case that all if we were to use a different sub-command the set of options would be different and therefore we do not need to make the options bear a tool-specific name?

Yes, with the set-up in this PR you can have `dbt` specific flags, with the "old" set-up this was not possible. Therefore I decided to remove the `dbt` prefix, since it is redundant now.

>One thing though, is that since we've already released this with the old names, not offering support for the old name would be a breaking change, which we don't want. I'm pretty sure there's a way to make click accept different versions of an option that map onto the same one right? If so, would we temporarily allow for that? Then What I would do is throw a deprecation warning that --dbt- CLI options will stop being supported in version x and make sure that we deprecate it in said version.

Ok, true, but wasn't it a "pre-release" for dbt ingestion. Therefore it is ok to change the flags, but this is ultimately a decision up to you - or Vijay.

Reply:
Don't get me wrong, I like and think that the renaming makes sense given that we don't need to specify `dbt` anymore.

The issue I have is that this dbt functionality is in a non pre-release tag. We may say it's pre-release but not according to semver. So, at the very least I would just offer the two as possibilities via:

```python
@click.option("--dbt-artifacts", "artifacts")
```

And then later, we can decide to just deprecate the one that we think is silly.

That being said, I would like @vijaykiran to make a call on this.